doc/getting_started/setup_fpga.md

```diff
@@ -549,15 +553,15 @@ First, make sure the device software has been built with debug symbols (by defau
 For example, to build and test the UART smoke test with debug symbols, you can add `--copt=-g` flag to the `./bazelisk.sh test ...` command:
 ```sh
 cd $REPO_TOP
-./bazelisk.sh test --copt=-g --test_output=streamed //sw/device/tests:uart_smoketest_fpga_cw340_rom_with_fake_keys
+./bazelisk.sh test --copt=-g --test_output=streamed //sw/device/tests:uart_smoketest_fpga_${BOARD}_rom_with_fake_keys
```

Comment:
The rest of the examples that are updated use `cw340` instead of `${BOARD}` so this should stay consistent.